storage/flux/table.go

```diff
@@ -14,6 +14,14 @@ import (
 	"github.com/influxdata/influxdb/v2/models"
 )
 
+func offset(t, window int64) int64 {
```

Comment:
```suggestion
func offset(t, windowEvery int64) int64 {
```

I feel that's more consistent and clear to other people?